shared/cfg/guest-hw.cfg

```diff
@@ -181,13 +180,9 @@ variants:
         # Mellanox CX4 MT27700 Family
         # For PowerPC driver (kernel module) that supports SR-IOV hardware is
         # mlx5_core
-        driver = igb
+        #driver = igb
```

Comment:
I comment it because the driver is different for different pfs.  Since several kinds of pfs can be on one host at same time, we want to test them by running avocado only once, just like https://github.com/autotest/tp-qemu/pull/1026#issuecomment-318313569.